subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -47,6 +47,21 @@ For example, the use of deprecated APIs in Kotlin DSL will emit warnings each ti
 
 This is a potentially breaking change if you are consuming the console output of Gradle builds.
 
+==== Gradle caches may contain CACHEDIR.TAG files
```

Comment:
I don't think we should categorize this as a potential breaking change.  This shouldn't break anyone's builds.  We can't be responsible for everything else going on on a user's machine.

Reply:
Yeah, I was just unsure about making this its own category. I'll move it up a level.